qiskit/user_config.py

```diff
@@ -138,6 +139,67 @@ def read_config_file(self):
                 self.settings['num_processes'] = num_processes
 
 
+def set_config(key, value, section=None, file=None):
```

Comment:
Can we name this `filename` or `file_path`instead of `file` just to make it clear that it's not a file (or file-like) object but actually a path.